test/lib/next-modes/base.ts

```diff
@@ -207,8 +207,8 @@ export class NextInstance {
         const finalDependencies = {
           react: reactVersion,
           'react-dom': reactVersion,
-          '@types/react': 'latest',
-          '@types/react-dom': 'latest',
+          '@types/react': '^19.1.1',
+          '@types/react-dom': '^19.1.2',
```

Comment:
CI prefers offline so we need to bump the version here as well to ensure isolated tests actually get the updated version